src/gui/FileBrowser.cpp

```diff
@@ -703,6 +752,25 @@ QList<QAction*> FileBrowserTreeWidget::getContextActions(FileItem* file, bool so
 	return result;
 }
 
+void FileBrowserTreeWidget::addFavorite(QString item)
+{
+	item = ConfigManager::removeTrailingSeparators(item);
+	ConfigManager::addFavoriteItem(item);
+	FileBrowser::getFavoritesBrowserInstance()->reloadTree();
+}
```

Comment:
Editing the passed in `item` directly feels unusual. This is similar to the suggestions I've made about the newly added functions in `ConfigManager`.
```suggestion
void FileBrowserTreeWidget::addFavorite(const QString& path)
{
	ConfigManager::addFavoriteItem(item);
	FileBrowser::getFavoritesBrowserInstance()->reloadTree();
}
```